clearwater_cassandra/cassandra_failed_plugin.py

```diff
@@ -54,10 +54,14 @@ def in_sig_namespace(command):
         # We must remove the node from the cassandra cluster. Get the node's ID
         # from nodetool status, then remove it with nodetool remove
         try:
-            status_command = "nodetool status | grep " + self._ip
-            output = subprocess.check_output(in_sig_namespace(status_command),
-                                             shell=True,
-                                             stderr=subprocess.STDOUT)
+            process_nodetool = subprocess.Popen(shlex.split(in_sig_namespace('nodetool status')),
```

Comment:
This isn't the correct way to do this. We should be doing:

`subprocess.Popen(['/usr/share/clearwater/bin/run-in-signaling-namespace', 'nodetool', 'status'])` instead of using `shlex.split`.

The entire point of run_in_sig_namespace seems dubious to me given run_command takes a namespace parameter....